cmd/snap/inhibit.go

```diff
@@ -116,13 +128,15 @@ func waitWhileInhibited(ctx context.Context, cli *client.Client, snapName string
 	}
 
 	if notified {
-		if err := flow.FinishInhibitionNotification(ctx); err != nil {
-			hintFlock.Close()
-			return nil, nil, nil, err
-		}
+		go func() {
+			if err := flow.FinishInhibitionNotification(ctx); err != nil {
```

Comment:
assuming that Finish.. doesn't do anything interesting, and our only flow implementation's finish code is nop, we could drop it.